src/ImpactEvaluator.sol

```diff
@@ -6,9 +6,8 @@ pragma solidity ^0.8.19;
 contract ImpactEvaluator is AccessControl {
     struct Round {
         uint end;
-        address payable[] addresses;
-        uint64[] scores;
         bool exists;
+        uint totalScores;
```

Comment:
we don't need to keep track of addresses and scores per round, but we track the sum of scores submitted, to be able prevent paying out more than expected, when too high a sum of scores has been submitted